### ISSUE TITLE
warning: Attempted to import the module

### ISSUE THREAD

**Issue by steida** [paulmillr/noble-hashes#76]
When noble-hashes are used with React Native

`warning: Attempted to import the module "/Users/danielsteigerwald/dev/evolu/node_modules/@noble/hashes/crypto.js" which is not listed in the "exports" of "/Users/danielsteigerwald/dev/evolu/node_modules/@noble/hashes". Falling back to file-based resolution. Consider updating the call site or asking the package maintainer(s) to expose this API.
`

**Comment by paulmillr** [comment]
can you show the exact import of noble that you are using?

**Comment by steida** [comment]
```ts
import { hmac } from "@noble/hashes/hmac";
import { sha512 } from "@noble/hashes/sha512";
import { randomBytes } from "@noble/hashes/utils";
```

**Comment by paulmillr** [comment]
it's just a warning? is everything working?

**Comment by steida** [comment]
Yeah, it works. Maybe there's something wrong with Metro. I would wait for the official release - https://metrobundler.dev/docs/package-exports

**Comment by paulmillr** [comment]
i don't think we can do anything here. Crypto is declared in exports.
